docs/global-installation.md

```diff
@@ -8,11 +8,13 @@ title: Global Installation
 
 | Platform  | Install command
 |-----------|-------------------------------------------------------------------------------------------------
-| **macOS**     | `npm install -g reason-cli@3.1.0-darwin`
+| **macOS**     | `npm install -g reason-cli` or `npm install -g reason-cli@3.1.0-darwin`
```

Comment:
I don't think we should advertise `npm install -g reason-cli` but instead fix `latest` release tag not to point to Darwin build.

Reply:
what would latest point to?

Reply:
Either:
- a dummy package which errors with a message to install a correct package
- a dummy package which installs the correct package during a postinstall phase (cc @jordwalke why we didn't do this, I can't remember)